src/Flash/Flash.tsx

```diff
@@ -61,7 +63,7 @@ const StyledFlash = styled.div<StyledFlashProps>`
     margin-bottom: 0;
   }
 
-  svg {
+  [data-component='leadingVisual'] {
```

Comment:
This will break the layout of existing usages of Flash where icons relied on this style.
What's the plan to smoothly roll this out? Are we expected to fixing all Flash usages across the consumer codebase at the same time that we upgrade to this version? If so, how do we track this?

Reply:
I've marked this as a breaking change, so it wouldn't go out until the next major release. We'd either handle the migration in dotcom ourselves or instruct folks on how to handle it themselves.